migrate.py

```diff
@@ -273,6 +273,8 @@ def rewrite_imports_in_fst(mod_fst, import_map, collection, spec):
             continue  # Skip imports of Base classes
         if len(imp.find_all('name_as_name', value='g:*loader*')) > 0:
             continue  # Skip imports of ansible.plugin.loader.py
+        if len(imp.find_all('name_as_name', value='g:AnsiblePlugin')) > 0:
```

Comment:
is there a way we can pass a list to value vs creating a bunch of if statements?

Reply:
I looked earlier and didn't find a way, but now that I am looking again, apparently there is https://redbaron.readthedocs.io/en/latest/querying.html#list-or-tuple. Let me test and update the PR.